terralab/commands/pipelines_commands.py

```diff
@@ -22,8 +23,10 @@ def list():
     LOGGER.info(
         f"Found {len(pipelines_list)} available pipeline{'' if len(pipelines_list) == 1 else 's'}:"
     )
+
     for pipeline in pipelines_list:
-        LOGGER.info(f"- {pipeline.pipeline_name} ({pipeline.description})")
+        LOGGER.info(pipeline.pipeline_name)
+        LOGGER.info(indented(pipeline.description))
```

Comment:
any particular reason we use tabs here vs a pad?  I can think of some but was just curious.

Reply:
the `indented` function actually uses a set number of spaces. the full tab width to my eye looked too big. you could also use `pad_column("", 2)` but `indented` is simpler. don't have strong feelings though if this is confusing.

Reply:
gotcha that makes sense. we can poke at it as we develop more and get more use cases